apps/gallery/js/ImageEditor.js

```diff
@@ -1656,6 +1645,22 @@ ImageEditor.prototype.getCropRegion = function() {
   };
 };
 
+// Covert the cropped region in image pixels to object with left, right, top,
+// bottom properties in CSS Pixel to be used to show crop overlay
+ImageEditor.prototype.getCropOverlayRegion = function(region) {
+  var self = this;
+  var dpr = window.devicePixelRatio;
+
+  var cropOverlay = {};
+  cropOverlay.left = Math.floor((region.x * self.scale) / dpr);
+  cropOverlay.top = Math.floor((region.y * self.scale) / dpr);
+  cropOverlay.right = Math.floor(((region.x + region.w) *
+                                 self.scale) / dpr);
+  cropOverlay.bottom = Math.floor(((region.y + region.h) *
+                                  self.scale) / dpr);
+  return cropOverlay;
+};
```

Comment:
Given that this function only gets called once, it seems like there ought to be some refactoring where it did not need to be a public method like this. But I can't think of how to do that cleanly, so a single-use function like this is the best we can do for now.